src/main/java/org/owasp/webgoat/container/MvcConfiguration.java

```diff
@@ -196,6 +199,25 @@ public Language language(LocaleResolver localeResolver) {
         return new Language(localeResolver);
     }
 
+    @Bean
+    public LocaleResolver localeResolver() {
+        SessionLocaleResolver localeResolver = new SessionLocaleResolver();
+        //localeResolver.setDefaultLocale(Locale.ENGLISH);
```

Comment:
Default is English, we can remove the comment?

Reply:
I will test if it is required, I don’t think so